### PR TITLE
bug.yml: update issue template

### PR DESCRIPTION
**Pull request by carlocab**
Another attempt at https://github.com/Homebrew/homebrew-core/pull/75049.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -14,9 +14,11 @@ body:
       required: true
   - type: checkboxes
     attributes:
-      description: Please verify that you've followed these steps
+      description: Please verify that you've followed these steps.
       options:
-        - label: The `brew doctor` above contains no "Warning" lines.
+        - label: I did `brew update` and am still able to reproduce my issue.
```
Comment (MikeMcQuaid) [inline .github/ISSUE_TEMPLATE/bug.yml:19]:
```suggestion
        - label: I ran `brew update` and am still able to reproduce my issue.
```

**Review by MikeMcQuaid — DISMISSED** [review]
Thanks!

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -14,9 +14,11 @@ body:
...
+          required: true
+        - label: Resolving the warnings from `brew doctor` did not fix my problem.
```
Comment (MikeMcQuaid) [inline .github/ISSUE_TEMPLATE/bug.yml:21]:
```suggestion
        - label: I have resolved all warnings from `brew doctor` and that did not fix my problem.
```

Would be good to get these into Homebrew/brew after this is merged, too.

Comment (carlocab) [reply .github/ISSUE_TEMPLATE/bug.yml:21]: Will do.

**Pull request merged**